.github/workflows/coverage_test.yaml

```diff
@@ -16,7 +19,21 @@ jobs:
             - name: Get submodules
               run: git submodule update --init --force --recursive
             - name: Install packages
-              run: sudo apt-get update && sudo apt-get install build-essential libhdf5-dev lcov
+              run: sudo apt-get update && sudo apt-get install build-essential libhdf5-dev
+            - name: Install lcov
+              run: |
+                VERSION="1.15"
+                wget "https://github.com/linux-test-project/lcov/releases/download/v${VERSION}/lcov-${VERSION}.tar.gz"
```

Comment:
huh, I wonder why this worked on my `g++ (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0` system

Reply:
Really ? Because it failed on both my PC and actions with gcc>7 (ubuntu18 tho). Do you have a lcov1.15 on ubuntu20 ?

Reply:
I have:
```
lcov: LCOV version 1.14
```
IIRC, the github actions runner's `latest` tag for ubuntu is 20.04, so maybe we should move to that, if it removes these headaches.

It can be done in another PR.

Reply:
Yeah I think we should keep it like this atm since it is done and check afterwards